docs/starters.yml

```diff
@@ -2383,3 +2383,16 @@
     - Uses MDX with Gatsby theme for quick and easy set up
     - Material-ui design with optional config passed into the theme options
     - Gradient background with sitemap, rss feed, and offline capabilities
+- url: https://gatsby-starter-sass.netlify.com/
+  repo: https://github.com/colbyfayock/gatsby-starter-sass
+  description: A Gatsby starter with Sass and no assumptions!
+  tags:
+    - Sass
```

Comment:
There is no `Styles` tag (see https://www.gatsbyjs.org/starters/?v=2 and "categories"). Could you change those to `Styling:Sass`, `Styling:SCSS` and remove `CSS` and `Styles` tags?

Reply:
`Styling:SCSS` is pretty much the only one needed here

Reply:
didn't occur to me those tags were the categories 🤦‍♂it's fixed now. I would have added `Styling:Sass` too, but that doesnt seem to exist in the current list